src/en/charms-storage.md

```diff
@@ -1,4 +1,5 @@
-Title: Using storage with Juju charms
+Title: Using Juju Storage
+TODO:  bug tracking: https://pad.lv/1708212
 
 # Using Juju Storage
```

Comment:
I think so. All VMs come with some sort of storage and so just sticking with "Juju Storage" sets good context.